config/initializers/github_client.rb

```diff
@@ -24,7 +24,7 @@ def self.enterprise?
 
   def self.github_url
     if enterprise?
```

Comment:
Hi @favonia 👋Thank you so much for contributing!

Looking at this PR, I think what you are trying to do is this: 

```suggestion
     if enterprise? && Rails.application.secrets.github_enterprise_url.present?
```

Reply:
I do not think this is necessary. `enterprise?` returns `true` only when `Rails.application.secrets.github_enterprise_url.present?` return trues. Please tell me if I was misguided.

https://github.com/education/classroom/blob/6561b4630e45e5455fdda6f479cde2ec6aaf54d5/config/initializers/github_client.rb#L19-L23

Reply:
Confirmed this is a bug and great catch, thanks @favonia! I'll work on a regression test!

Reply:
Actually in digging into this more, I don't expect it to regress and I think your fix is great.